game/missiongenerator/forcedoptionsgenerator.py

```diff
@@ -41,6 +41,10 @@ def _set_labels(self) -> None:
             self.mission.forced_options.labels = ForcedOptions.Labels.NeutralDot
         elif self.game.settings.labels == "Off":
             self.mission.forced_options.labels = ForcedOptions.Labels.None_
+        elif self.game.settings.labels == "Full":
+            self.mission.forced_options.labels = ForcedOptions.Labels.Full
+        else:
+            self.mission.forced_options.labels = ForcedOptions.Labels.Default
```

Comment:
Else just do nothing, right? The check is failing because there is no `ForcedOptions.Labels.Default`.